src/App.tsx

```diff
@@ -1,63 +1,39 @@
 import * as React from "react";
-import Container from "@mui/material/Container";
-import Typography from "@mui/material/Typography";
-import Box from "@mui/material/Box";
-import Link from "@mui/material/Link";
-import ProTip from "./ProTip";
-import { render } from "react-dom";
+import { ThemeProvider } from "@material-ui/core";
```

Comment:
As we continued to add more and more MUI components we did not seem to see any issues with using the old ThemeProvider. However, in the interest of limiting our packages to what we need, as well as to avoid any possible future issues, we'll change all imports to those from `@mui/material`